src/lib/PointerTracker/index.ts

```diff
@@ -172,19 +187,18 @@ export class PointerTracker {
     const changedPointers = ('changedTouches' in event) ? // Shortcut for 'is touch event'.
       Array.from(event.changedTouches).map(t => new Pointer(t)) :
       [new Pointer(event)];
-
-    let shouldCallback = false;
+    const trackedChangedPointers = [];
 
     for (const pointer of changedPointers) {
       const index = this.currentPointers.findIndex(p => p.id === pointer.id);
-      if (index === -1) continue;
-      shouldCallback = true;
+      if (index === -1) continue; // Not a pointer we're tracking
+      trackedChangedPointers.push(pointer);
       this.currentPointers[index] = pointer;
     }
 
-    if (!shouldCallback) return;
+    if (!trackedChangedPointers[0]) return;
```

Comment:
`trackedChangedPointers.length <= 0`?

Reply:
I wouldn't use `<` unless it could be less than 1, but happy to go for `=== 0`.